docs/wiki/Whats-new.md

```diff
@@ -51,6 +51,8 @@ This article will be updated as and when changes are made to the above and anyth
 Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
 ### TBD 2023
+- Add Policy initiative [Unused resources driving cost should be avoided (azadvertizer.net)](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/Audit-UnusedResourcesCostOptimization.html)
+  - Added a Policy Assignment `Audit-UnusedResourcesCostOptimization` ([Unused resources driving cost should be avoided (azadvertizer.net)](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/Audit-UnusedResourcesCostOptimization.html)) to audit unused resources that are driving cost at the intermediate root management group
```

Comment:
You can remove this section, if you agree with my comment on the initiative.

Reply:
Im good with this if the above is merged. Keeps it cleaner